janus_core/cli/train.py

```diff
@@ -52,4 +61,15 @@ def train(
     elif "foundation_model" in config:
         raise ValueError("Please include the `--fine-tune` option for fine-tuning")
 
-    run_train(mlip_config)
+    inputs = {"mlip_config": str(mlip_config), "fine_tune": fine_tune}
+
+    # Save summary information before training begins
+    start_summary(command="train", summary=summary, inputs=inputs)
```

Comment:
Would this be better implemented as a context? Not for this PR though

Reply:
Potentially. There is a [context manager](https://mlco2.github.io/codecarbon/usage.html#context-manager), and function decorators too, which would be simpler, but I didn't think they integrated well with saving to the log, or having individual tasks when we have calculations that do multiple things, when I originally implemented this. Worth revisiting at some point